File: runtimes/nodejs/actions/repackage-message.js
var request = require('request');
var request_promise = require('request-promise');

/*
  Takes information from a github event and puts it in a form consumable by
  the whisk.system/slack/post action.
*/
function main(params){
  var message = `commit ${params.head_commit.id} was pushed to ${params.repository.full_name}`;
  return {text: message}
}

File: runtimes/swift/actions/repackage-message.swift
func main(params: [String:Any]) -> [String:Any] {
  if let head_commit = params["head_commit"] as? [String:Any] , let repository = params["repository"] as? [String:Any]{
  	if let id = head_commit["id"] , let fullName = repository["full_name"]{
  		print(id)
          print(fullName)
          let message = "commit \(id) was pushed to \(fullName)"
          return ["text": message]
  	}
  }
  return ["text":"Error occurred when attempting to parse information regarding latest git push event."]
}
